src/cascadia/inc/cppwinrt_utils.h

```diff
@@ -185,3 +185,27 @@ std::vector<wil::com_ptr<T>> SafeArrayToOwningVector(SAFEARRAY* safeArray)
 
     return result;
 }
+
+#define DECLARE_CONVERTER(nameSpace, className)                                                                   \

```

Comment:
Might as well use `DECLARE_CONVERTER` on all the other converter header files. For consistency.